zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```diff
@@ -954,6 +1203,62 @@ components:
           type: array
           items:
             $ref: "#/components/schemas/CamundaUserResponse"
+    CamundaGroupRequest:
+      type: "object"
+      properties:
+        id:
+          type: "integer"
+          format: "int64"
+        name:
+          type: "string"
+    CamundaGroupResponse:
+      type: "object"
+      properties:
+        id:
+          type: "integer"
+          format: "int64"
+        name:
+          type: "string"
+    GroupSearchResponse:
+      type: object
+      allOf:
+        - $ref: "#/components/schemas/SearchQueryResponse"
+      properties:
+        items:
+          type: array
+          items:
+            $ref: "#/components/schemas/CamundaGroupResponse"
+    FilterDto:
+      type: "object"
+      properties: { }
+    SortDto:
+      type: "object"
+      properties: { }
+    PageDto:
+      type: "object"
+      properties: { }
+    SearchRequestDto:
+      type: "object"
+      properties:
+        filter:
+          type: "object"
+          properties: { }
+        sort:
+          type: "object"
+          properties: { }
+        page:
+          type: "object"
+          properties: { }
+    GroupSearchResponseDto:
```

Comment:
❌ For consistency, please avoid `...Dto` names. Having `...Request` and `...Response` in the names should be meaningful enough already.